plugins/lookup/onepassword.py

```diff
@@ -265,6 +269,7 @@ def run(self, terms, variables=None, **kwargs):
         section = kwargs.get('section')
         vault = kwargs.get('vault')
         op.subdomain = kwargs.get('subdomain')
+        op.domain = kwargs.get('domain', '1password.com')
```

Comment:
Looks like this should be converted to using the config manager (`set_options`/`get_option`) eventually :)